src/hevm/src/EVM/SymExec.hs

```diff
@@ -335,9 +335,15 @@ simplify e = if (mapExpr go e == e)
     go (CopySlice (Lit 0x0) (Lit 0x0) (Lit 0x0) _ dst) = dst
 
     -- simplify buffers
+    go o@(ReadWord a (WriteWord b c _))
```

Comment:
Not necessarily opposed to this rule, but curious it when we have Expr.readWord?

Reply:
The function selection(?) seems to produce this:

```
            (ITE (Eq (ReadWord (Lit 0x4) (Fact (PGEq (BufLength (WriteWord (Lit 0x4) (Fact (PAnd (PGEq (Var "arg1") (Lit 0x0)) (PLEq (Var "arg1") (Lit 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff))) (Var "arg1")) (WriteByte (Lit 0x0) (LitByte 182) (WriteByte (Lit 0x1) (LitByte 181) (WriteByte (Lit 0x2) (LitByte 95) (WriteByte (Lit 0x3) (LitByte 37) (AbstractBuf "txdata"))))))) (Lit 0x24)) (WriteWord (Lit 0x4) (Fact (PAnd (PGEq (Var "arg1") (Lit 0x0)) (PLEq (Var "arg1") (Lit 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff))) (Var "arg1")) (WriteByte (Lit 0x0) (LitByte 182) (WriteByte (Lit 0x1) (LitByte 181) (WriteByte (Lit 0x2) (LitByte 95) (WriteByte (Lit 0x3) (LitByte 37) (AbstractBuf "txdata")))))))) (Lit 0x0))
```